src/Task/AbstractTask.php

```diff
@@ -66,6 +66,8 @@ protected function defineOptions(array $options = [], array $additionalOptions =
             'execution_end_time' => null,
             'last_execution' => null,
             'max_duration' => null,
+            'max_execution' => null,
+            'max_retry' => null,
```

Comment:
What about using plural? (max_executions & max_retries)

Also applies to getters, setters & $resolver->setInfo()

Reply:
Agree, improved